api/api/clubs/urls.py

```diff
@@ -4,5 +4,7 @@
 
 urlpatterns = [
     path('', views.ListClubsView.as_view()),
-    path('<int:pk>/', views.UpdateClubsView.as_view())
+    path('<int:pk>/', views.UpdateClubsView.as_view()),
+    path('<int:pk>/register/', views.CreateClubRegistrationRequestView.as_view()),
+    path('registration/<int:pk>/', views.UpdateClubRegistrationRequestView.as_view())
```

Comment:
url should be `<int:pk>/registration-requests/<int:pk`

Reply:
Again don't understand the need for `<int:pk>/registration-request<...>` . I suppose we only need registration id to update a particular request .

Reply:
This change will only affect the URL ? coz current implementation deals only with `registration id`

Reply:
Registration request is a child resource of club. So in the implementation find should be called with club id and registration request id